_pages/interviews/consulting-engineer.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Consulting Engineer Interview Guide
+parent: interviews
+---
+
+*This is a supplement to the [technical engineering interview guide]({{ site.baseurl }}/interviews/engineer/). Please read that guide first. The questions here can be added to the technical interview for Consulting Engineer candidates.*
+
+### Consulting Engineer Questions
+
+* [“Please explain a complex technical concept or tool to me as if I were a non-technical partner](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl)
```

Comment:
I really like this.

Reply:
Hat tip to @mgwalker thanks again for the suggestion

Reply:
This is delightful. Minor copy edit: remove the open-quote from the link text for consistency with the other questions (`"Please explain` --> `Please explain`).

Reply:
Removed `"` . Thanks!